src/main/proto/data_transfer_objects.proto

```diff
@@ -641,16 +642,18 @@ enum TokenUpdateDto {
     setPortraitImage = 43;
     setCharsheetImage = 44;
     setLayout = 45;
-    clearLightSources = 46;
-    removeLightSource = 47;
-    addLightSource = 48;
-    setHasSight = 49;
-    setSightType = 50;
-    flipX = 51;
-    flipY = 52;
-    flipIso = 53;
-    setSpeechName = 54;
-    removeFacing = 55;
+    createUniqueLightSource = 46;

```

Comment:
Because backwards compatibility isn't an issue (client + server always have to match anyways) I'd rather make it read nicely. If/when BC concerns change in the future, we'll of course be more particular.

And if there is some 3rd party tool that also consumes our protobufs, it would be that author's responsibility to keep things up-to-date until we commit to a stable protocol.